kong/cmd/utils/prefix_handler.lua

```diff
@@ -414,5 +469,6 @@ return {
   compile_kong_conf = compile_kong_conf,
   compile_kong_stream_conf = compile_kong_stream_conf,
   compile_nginx_conf = compile_nginx_conf,
-  gen_default_ssl_cert = gen_default_ssl_cert
+  gen_default_ssl_cert = gen_default_ssl_cert,
+  gen_default_dhparams = gen_default_dhparams,
```

Comment:
It seems that `gen_default_dhparams` is only called from this file (and is most likely only useful in this file), is there a reason for exporting it?